docs/source/release/v6.1.0/mantidworkbench.rst

```diff
@@ -8,52 +8,64 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 
+General
+#######
 - Added Floating/On Top setting for all the windows that are opened by workbench (plots, interfaces, etc.)
-- New plot interactions: Double click a legend to hide it, double click a curve to open it in the plot config dialog.
-- It is now possible to overplot bin data from the matrix workspace view.
 - New command line options ``--version`` will print the version on mantid and exit. ``--error-on-warning`` will convert python warnings into exceptions. This is intended for developers so they can find deprecation warnings more easily.
+- Calling python's `input` now raises an input dialog in the script editor and the iPython shell.
 - Improved the performance of the table workspace display for large datasets
 - Added a sample material dialog that is accessed via the context menu in the workspace widget.
-- When a workspace is renamed it now updates the relevant plot labels with the new workspace name.
-- Add a checkbox to freeze the rotation in the instrument viewer in Full 3D mode.
-- Calling python's `input` now raises an input dialog in the script editor and the iPython shell.
-- A new empty facility with empty instrument is the default facility now, and
-  user has to select their choice of facility (including ISIS) and instrument for the first time.
-- Added an algorithm ProfileChiSquared1D to profile chi squared after a fit. This can be used
-  to find better estimates of parameter errors.
-- Instrument view: when in tube selection mode, the sum of pixel counts is now output to the selection pane.
-- Added legend property commands to the plot script generator.
+- A new empty facility with empty instrument is the default facility now, and  user has to select their choice of facility (including ISIS) and instrument for the first time.
 - Added memory widget to display total memory usage. This means that your widget layout will be reset when starting workbench v6.1.0 for the first time. Previously saved layouts accessible from ``View > User Layouts`` may need to be saved again to include the memory bar widget.
```

Comment:
I'll repeat these comments on the proper PR so you are bombarded in the same place. It would be nice for the memory widget note to link to this page: https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/getting_started/main_window.html